### PR TITLE
Projectkk2glider/simu tmp sd path

### PR DESCRIPTION
**Pull request by projectkk2glider**
Not ready for merge yet! Just to show you if I am going in the right direction.

This addresses first item of #4224

### CODE REVIEW

**Comment by projectkk2glider** [comment]
With this in place and running simu like this: `simu aaa /card /tmp` will:
* use `/card` path as the root of the SD card image
* any path starting with `/RADIO/...` or `/MODELS/...` will be mapped into `/tmp/RADIO/...` and `/tmp/MODELS/...`
* the SD browser will still browse `RADIO` and `MODEL` directories from `/card` path. To change this will be difficult and in my opinion unnecessary.

**Comment by projectkk2glider** [comment]
Going so deep would be a problem, especially the f_readdir() would be difficult to implement. Not sure if it is needed. For simulation I think the current state is enough - the radio/model settings get read and written for the alternate path. 

Basically anything that uses absolute path will be remapped correctly. What does not work is:
 * iterating trough all files in a root of SD card (won't show RADIO and MODELS dirs from alternate path)
 * changing directories with f_cwd() using relative paths (again it wont go into the alternate dirs for RADIO and MODELS)

**Comment by bsongis** [comment]
Yes it's perfect as it is now

File: radio/src/targets/simu/simufatfs.cpp [hunk 1]
```diff
@@ -0,0 +1,554 @@
...
+void splitPath(const std::string & path, std::string & dir, std::string & name)
+{
+#if defined(WINDOWS_BUILD)
+  char drive[_MAX_DRIVE];
+  char dir[_MAX_DIR];
+  char fname[_MAX_FNAME];
+  char ext[_MAX_EXT];
+  _splitpath(path.c_str(), drive, dir, fname, ext);
+  name = std::string(fname) + std::string(ext);
+  dir = std::string(drive) + std::string(dir);
```
Comment (kilrah) [inline radio/src/targets/simu/simufatfs.cpp]: Compilation broken here on VS:

```20>C:\opentx-22\code\radio\src\targets\simu\simufatfs.cpp(135): error C2082: redefinition of formal parameter 'dir'
20>C:\opentx-22\code\radio\src\targets\simu\simufatfs.cpp(138): error C2664: 'void _splitpath(const char *,char *,char *,char *,char *)': cannot convert argument 3 from 'std::string' to 'char *'
20>  C:\opentx-22\code\radio\src\targets\simu\simufatfs.cpp(138): note: No user-defined-conversion operator available that can perform this conversion, or the operator cannot be called
20>C:\opentx-22\code\radio\src\targets\simu\simufatfs.cpp(141): error C2065: 'dirName': undeclared identifier
20>C:\opentx-22\code\radio\src\targets\simu\simufatfs.cpp(187): error C2065: 'fileName': undeclared identifier
20>C:\opentx-22\code\radio\src\targets\simu\simufatfs.cpp(187): error C2228: left of '.c_str' must have class/struct/union
20>  C:\opentx-22\code\radio\src\targets\simu\simufatfs.cpp(187): note: type is 'unknown-type' ```

Comment (projectkk2glider) [inline radio/src/targets/simu/simufatfs.cpp]: I have no Windows build environment. Can you fix this?

Comment (projectkk2glider) [inline radio/src/targets/simu/simufatfs.cpp]: Hopefully fixed by 19213c8

**Pull request merged**